util-flywayinstaller-cli/pom.xml

```diff
@@ -62,28 +62,10 @@
                     <archive>
                         <manifest>
                             <addClasspath>true</addClasspath>
```

Comment:
Why are we still adding the classpath to the manifest if we're not setting the main class?